src/libstd/sync/mpsc/mod.rs

```diff
@@ -1267,6 +1267,38 @@ impl error::Error for TryRecvError {
     }
 }
 
+#[stable(feature = "mpsc_recv_timeout_error", since = "1.13.0")]
+impl fmt::Display for RecvTimeoutError {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
```

Comment:
Can't this be reduced now to `self.description().fmt(f)`? Though, I would think you'd want to write something like `RecvTimeoutError(<message here>)` instead of the bare message.

I notice that both of these comments also apply to `std::sync::mpsc::{RecvError, TryRecvError}` so maybe they are wrong.